packages/http-service/README.md

```diff
@@ -0,0 +1,31 @@
+# HTTP Service
+
+This package wraps connections managed by the `connection-manager` package into an Observable.
+
+## Usage
+```javascript
+import { request, stream } from "http-service";
+
+request("http://localhost:4200/payload.json")
+  .retry(3)
```

Comment:
@nLight @orlandohohmeier this isn't how we're going to use retry right?

Reply:
We are free to use the built in retry with `http-service`, but you are correct we're going to implement the back-off retry. Just not for now.